### PR TITLE
Add glossary entry for "Term".

### LINKED ISSUE #672: Please add "Raft Term" to the Glossary
Please add "Term" or "Raft Term" to the glossary at https://etcd.io/docs/v3.5/learning/glossary/

Thank you.

### PR DESCRIPTION
**Pull request by guettli**
closes #672

### CODE REVIEW

File: content/en/docs/v3.6/learning/glossary.md [hunk 1]
```diff
@@ -88,6 +88,11 @@ A point-in-time backup of the etcd cluster state.
 
 The physical storage backing the cluster keyspace.
 
+## Term
+
+A term is a monotonically increasing integer that is associated with each leader election in the Raft algorithm.
```
Comment (serathius) [inline content/en/docs/v3.6/learning/glossary.md:93]: Would be good to mention that for "For a term there can only be one elected leader and term is incremented on leader change". 

I think this is accurate +/- pre-vote and I think that leader election can fail so there might not be a leader for a term.

Comment (guettli) [reply content/en/docs/v3.6/learning/glossary.md:93]: @serathius  I added your suggestion and removed the second sentence. Does it make sense now?

**Comment by ahrtr** [comment]
Looks good to me. Thanks @guettli . Please squash the commits and also signoff the final commit.

**Comment by jmhbnz** [comment]
Hey @ahrtr, @serathius - I believe this contribution is ready to merge, can you please take a look?

**Comment by ahrtr** [comment]
Sorry for the late response. Thanks @guettli and @jmhbnz .

**Pull request merged**